src/assembler.jl

```diff
@@ -33,24 +40,28 @@ as described in the [manual](@ref man-assembly).
     the same pattern. See the [manual section](@ref man-assembly) on assembly.
 """
 function start_assemble(N::Int=0)
-    return Assembler(N)
+    return start_assemble(Float64, N)
+end
+
+function start_assemble(t::Type{T}, N::Int=0) where T
+    return COOAssembler(t, N)
```

Comment:
Just noticed, change to
```suggestion
function start_assemble(::Type{T}, N::Int=0) where T
    return COOAssembler(T, N)
```
?

Reply:
Well, too slow...

Reply:
Reworked in  #1058